deploy/mainnet/442_add_to_PoolRegistry.ts

```diff
@@ -0,0 +1,196 @@
+          typeOfAsset: PoolType.BTC,
+          poolName: ethers.utils.formatBytes32String("tBTCv2-BTCv2"),
+          targetAddress: (await get("MetaSwapUpdated")).address,
+          metaSwapDepositAddress: (
+            await get("SaddleTBTCMetaPoolUpdatedDeposit")
+          ).address,
+          isSaddleApproved: true,
+          isRemoved: false,
+          isGuarded: false,
+        },
+        {
+          // sUSD meta pool updated
+          poolAddress: (await get("SaddleSUSDMetaPoolUpdated")).address,
+          typeOfAsset: PoolType.USD,
+          poolName: ethers.utils.formatBytes32String("sUSD-USDv2"),
+          targetAddress: (await get("MetaSwapUpdated")).address,
+          metaSwapDepositAddress: (
+            await get("SaddleSUSDMetaPoolUpdatedDeposit")
+          ).address,
+          isSaddleApproved: true,
+          isRemoved: false,
+          isGuarded: false,
+        },
+        {
+          // WCUSD meta pool updated
+          poolAddress: (await get("SaddleWCUSDMetaPoolUpdated")).address,
+          typeOfAsset: PoolType.USD,
+          poolName: ethers.utils.formatBytes32String("WCUSD-USDv2"),
+          targetAddress: (await get("MetaSwapUpdated")).address,
+          metaSwapDepositAddress: (
+            await get("SaddleWCUSDMetaPoolUpdatedDeposit")
+          ).address,
+          isSaddleApproved: true,
+          isRemoved: false,
+          isGuarded: false,
+        },
+        {
+          // vETH2 pool
+          poolAddress: (await get("SaddleVETH2Pool")).address,
+          typeOfAsset: PoolType.ETH,
+          poolName: ethers.utils.formatBytes32String("vETH2"),
+          targetAddress: "0x5847f8177221268d279Cf377D0E01aB3FD993628",
+          metaSwapDepositAddress: ZERO_ADDRESS,
+          isSaddleApproved: true,
+          isRemoved: false,
+          isGuarded: false,
+        },
+        {
+          // BTC pool
+          poolAddress: (await get("SaddleBTCPool")).address,
+          typeOfAsset: PoolType.BTC,
+          poolName: ethers.utils.formatBytes32String("BTC"),
+          targetAddress: (await get("SaddleBTCPool")).address,
+          metaSwapDepositAddress: ZERO_ADDRESS,
+          isSaddleApproved: true,
+          isRemoved: true,
+          isGuarded: true,
+        },
+        {
+          // USD pool
+          poolAddress: (await get("SaddleUSDPool")).address,
+          typeOfAsset: PoolType.USD,
+          poolName: ethers.utils.formatBytes32String("USD"),
+          targetAddress: "0x98D2aFc66DE1F73598c6CFa35cbdfebB135fb8FA",
+          metaSwapDepositAddress: ZERO_ADDRESS,
+          isSaddleApproved: true,
+          isRemoved: true,
+          isGuarded: false,
+        },
+        {
+          // sUSD meta pool updated
+          poolAddress: (await get("SaddleSUSDMetaPool")).address,
+          typeOfAsset: PoolType.USD,
+          poolName: ethers.utils.formatBytes32String("sUSD-USDv2_outdated"),
```

Comment:
Good question. This doesnt seem like a good name since retroactively changing names is annoying to do. 

What do you think about

`sUSD-USDv2` (outdated one)
`sUSD-USDv2_v2`

as the two pool names?